sycl/test-e2e/Graph/Inputs/memadvise.cpp

```diff
@@ -0,0 +1,60 @@
+// Tests adding a USM mem advise operation as a graph node.
+
+#include "../graph_common.hpp"
+
+static constexpr int Count = 100;
+
+int main() {
+
+  using T = int;
+
+  queue Queue{{sycl::ext::intel::property::queue::no_immediate_command_list{}}};
+
+  if (!are_graphs_supported(Queue)) {
+    return 0;
```

Comment:
Nit; It would be nice to print something if this is hit, just so it doesn't just pass silently. It won't show up in LIT runs where it just pass, but at least it will if someone runs it in isolation.